src/components/tooltips-hoc/positionValidation.ts

```diff
@@ -0,0 +1,72 @@
+import {CSSProperties} from 'react';
+
+export const positionValidator: {[position: string]: (boxOffset: ClientRect | DOMRect, tooltipOffset: ClientRect | DOMRect, window: Window) => Partial<CSSProperties>} = {

```

Comment:
FIle should be named `postitionValidator` or `PositionValidator` (same name as exported const).